wikipedia-dark.user.css

```diff
@@ -1,7 +1,7 @@
 /* ==UserStyle==
 @name         Wikipedia Dark
 @namespace    StylishThemes
-@version      3.4.1
+@version      3.4.2
```

Comment:
Hello, please again, dont include release commits, these are intended for internal use only.

Reply:
Got it. 👍